receiver/udplogreceiver/README.md

```diff
@@ -0,0 +1,30 @@
+# UDP Receiver
+
+Receives logs from udp using
+the [opentelemetry-log-collection](https://github.com/open-telemetry/opentelemetry-log-collection) library.
```

Comment:
Please explain the protocol/usages since probably that udp library part will be replaced by this.

Reply:
Could you give some example please?
I'm not exactly sure what are you asking for